### ISSUE TITLE
IOC: JASCO 4180 - Reset fixed component selection state

### ISSUE THREAD

**Issue by aaron-long** [ISISComputingGroup/IBEX#4199]
While using the JASCO 4180 HPLC pump on ZOOM and CRISP, it was seen that the pump could get into a single component manual select mode. In this mode you can select only a single component at a time with 100% from the front panel. The device will not accept component set commands with mixed compositions (i.e., 50% component A, 50% component B).

It was found that the pump command with enum 8, which maps to a "Reset to initial conditions and rerun the program" fixed this state.

- [x] JASCO 4180 Can be reset from single component selection mode remotely.

**Comment by ThomasLohnert** [comment]
(Since this is likely the next ticket to touch the Jasco IOC) 
Consider doing https://github.com/ISISComputingGroup/IBEX/issues/4284 at the same time.

**Comment by aaron-long** [comment]
Changes made but waiting on the device being set up in the lab to test against.

Tickets #4198 and #4284 were also completed as they were trivial changes.

**Comment by aaron-long** [comment]
We attempted to replicate the "Fixed component mode" behaviour with different JASCO pumps in the Soft Matter  lab. We set up an brand new, and unused, pump to see if it initialised in this state. From the test it doesn't seem to do this though.

The changes made to the OPI and the new `RESET:SP` should give a clear indication to users on how to troubleshoot if they are unable to set the composition values.
